### PR TITLE
Misc1

### PR DESCRIPTION
**Pull request by kraiby30**

### CODE REVIEW

File: RT_SYCL/texture.hpp [hunk 1]
```diff
@@ -53,8 +53,7 @@ struct checker_texture {
         else
             return even.value(rec);
     }
-    solid_texture odd;
-    solid_texture even;
+    solid_texture odd, even;
 };
```
Comment (keryell) [inline RT_SYCL/texture.hpp:57]: Keep it separated so you can add a Doxygen comment describing each one

File: RT_SYCL/render.hpp [hunk 2]
```diff
@@ -111,43 +111,12 @@ class render_kernel {
...
-        vertical = focus_dist * viewport_height * v;
-        lower_left_corner = origin - horizontal / 2 - vertical / 2 - focus_dist * w;
-        auto lens_radius = 0.1 / aperature;
-
-        vec rd = lens_radius * random_in_unit_disk();
-        vec offset = u * rd.x() + v * rd.y();
-
-        return ray(origin + offset, lower_left_corner + s * horizontal + t * vertical - origin - offset);
-    }
-
```
Comment (keryell) [inline RT_SYCL/render.hpp:145]: So, all this is dead and now in camera.hpp?

File: RT_SYCL/camera.hpp [hunk 3]
```diff
@@ -86,6 +86,9 @@ class camera {
     lens_radius = aperture/2;
   }
 
+  /* Computes ray from camera passing through
```
Comment (keryell) [inline RT_SYCL/camera.hpp:89]: Use Doxygen style for it starting with `/**` so this comment can be captured as some documentation to generate in the future some pages like in https://trisycl.github.io/triSYCL/Doxygen/triSYCL/html/index.html

Look for example at typical triSYCL code for inspiration.
Tweak also the indent of the comment.

For more specific syntax: https://www.doxygen.nl/manual/docblocks.html

**Review by keryell — CHANGES_REQUESTED** [review]
A few nitpicks.

File: RT_SYCL/render.hpp (hunk 2 above)
Comment (kraiby30) [reply RT_SYCL/render.hpp:145]: Yea!

**Review by kraiby30 — COMMENTED** [review]
changed it in recent commit

File: RT_SYCL/camera.hpp [hunk 4]
```diff
@@ -86,6 +86,12 @@ class camera {
     lens_radius = aperture/2;
   }
 
+  /** get_ray(s,t) 
+     
+      Computes ray from camera passing through
```
Comment (keryell) [inline RT_SYCL/camera.hpp:91]: Do not repeat the source code
```suggestion
  /** Computes ray from camera passing through 
```

File: RT_SYCL/render.hpp (hunk 2 above)
Comment (keryell) [reply RT_SYCL/render.hpp:145]: Less source code means less potential bugs! :-)

File: RT_SYCL/texture.hpp [hunk 5]
```diff
@@ -53,7 +53,7 @@ struct checker_texture {
         else
             return even.value(rec);
     }
-    solid_texture odd;
+    solid_texture odd;
```
Comment (keryell) [inline RT_SYCL/texture.hpp:56]: Curious, why this trailing space?

**Review by keryell — CHANGES_REQUESTED** [review]
Almost there...

**Comment by kraiby30** [comment]
Shall I go ahead and merge this PR?

**Review by keryell — APPROVED** [review]
That looks good.
Thanks.

**Comment by keryell** [comment]
> Shall I go ahead and merge this PR?

It is better if the PR are not merged by its author when possible, for review quality purpose.

**Pull request merged**